code/code/sys/migrations.cc

```diff
@@ -143,6 +143,55 @@ void runMigrations() {
                     "value text not null, "
                     "foreign key (player_id) references player (id) on delete cascade)"));
         },
+        [&](){
+            vlogf(LOG_MISC, "Moving wiz data over to db");
+            assert(sneezy.query(
+                    "create table if not exists wizdata ("
+                    "setsev int not null,"
+                    "office int not null,"
+                    "blockastart int not null,"
```

Comment:
I can totally imagine an immortal with no office, or no assigned block. Why are they `not null`? Also, I'd add `default 0` to office.